scripts/nolints_prime_decls.txt

```diff
@@ -1615,6 +1615,7 @@ Finsupp.smul_single'
 Finsupp.subtypeDomain_eq_zero_iff'
 Finsupp.sum_apply'
 Finsupp.sum_cons'
+Finsupp.sum_insertNth'
```

Comment:
Please remove this. Either follow my suggestion above about renaming, or, if you disagree with that for some reason, add a docstring to this declaration.